docs/reference/language-reference/built-in-types/int.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Int"
+sidebar_label: "Int"
+sidebar_position: 2
+keywords:
+  - types
+  - integer
+  - unsigned
+  - number
+  - long
+draft: false
+---
+
+Integers are, by default, 64 bit signed integers.
+
+```darklang
+let x = 42
+```
+
+Other integer sizes supported are described below. To declare a variable with a
+specific integer size, append the size suffix to the value.
+
+| F# Equivalent | Description                    | Suffix | Examples          |
+| ------------- | ------------------------------ | ------ | ----------------- |
+| int8          | signed 8 bit number            | y      | `let fizz = 42y`  |
+| uint8         | unsigned 8 bit number          | uy     | `let buzz = 42uy` |
+| int16         | signed 16 bit number           | s      | `let fizz = 42s`  |
+| uint16        | unsigned 16 bit number         | us     | `let buzz = 42us` |
+| int32         | signed 32 bit number           | l      | `let fizz = 42l`  |
+| uint32        | unsigned 32 bit number         | ul     | `let buzz = 42ul` |
+| int64         | signed 64 bit number (default) | L      | `let fizz = 42`   |
```

Comment:
No suffix will be int64 as well.

Reply:
The example is meant to indicate that no suffix results in an int64. 

I could make it more obvious like:
`let fizz = 42` (no suffix)